plugins/blackduck-backend/README.md

```diff
@@ -10,7 +10,7 @@ This plugin needs to be added to an existing backstage instance.
 
 ```bash
 # From your Backstage root directory
-yarn add --cwd packages/backend backstage-plugin-blackduck-backend
+yarn --cwd packages/backend add backstage-plugin-blackduck-backend
```

Comment:
There is no necessary to keep `add` before the package name or after `yarn` command. It is just an reference of how to do. Its up to the adopter who they are going to include the package. 
So both works 
```bash
yarn --cwd packages/backend add backstage-plugin-blackduck-backend
```
```bash
yarn add --cwd packages/backend backstage-plugin-blackduck-backend
```

Reply:
That's odd that you say that because when I try to keep the add before the package name I get a usage error
![image](https://github.com/deepan10/backstage-plugins/assets/5551190/2596d804-2194-414e-a8b7-a446a8eb505b)
Additionally all backstage documentation I have seen has add statements formatted how I am proposing. 
[one example](https://backstage.io/docs/plugins/backend-plugin/#developing-your-backend-plugin): 
```
# From your Backstage root directory
yarn --cwd packages/backend add @internal/plugin-carmen-backend@^0.1.0 # Change this to match the plugin's package.json
```

Reply:
That's strange I verified multiple time with different version both are working even some community plugin uses the `add` just next to `yarn`.

`Backstage` official docs uses the same syntax in all places. Thanks for highlight, will go with the same syntax.